extensions/query-store/src/reports/baseQueryStoreReport.ts

```diff
@@ -101,6 +108,24 @@ export abstract class BaseQueryStoreReport {
 			CSSStyles: { 'margin-top': '5px', 'margin-bottom': '5px', 'margin-right': '15px' }
 		}).component();
 
+		// Open in New Tab button
+		const openInNewTabButton = view.modelBuilder.button().withProps({
+			label: constants.openInNewTab,
+			title: constants.openInNewTab,
+			iconPath: {
```

Comment:
Maybe worth bringing in the IconHelper class for this, since we'll have plenty more icons, I expect.